src/main/java/picard/analysis/CollectRnaSeqMetrics.java

```diff
@@ -168,7 +171,7 @@ protected void setup(final SAMFileHeader header, final File samFile) {
 
         collector = new RnaSeqMetricsCollector(METRIC_ACCUMULATION_LEVEL, header.getReadGroups(), ribosomalBasesInitialValue,
                 geneOverlapDetector, ribosomalSequenceOverlapDetector, ignoredSequenceIndices, MINIMUM_LENGTH, STRAND_SPECIFICITY, RRNA_FRAGMENT_PERCENTAGE,
-                true);
+                true, END_BIAS_BASES);
```

Comment:
This fixes the case inside picard itself, but if others are importing this code and depending on the original signature of the constructor, a backward compatible fix would be to overload the constructor and allow both forms; with and without the `END_BIAS_BASES` parameter.

Reply:
Hi @pshapiro4broad .  That comment makes sense around backwards compatibility.